designcompose/src/main/java/com/android/designcompose/FrameRender.kt

```diff
@@ -453,18 +463,36 @@ internal fun ContentDrawScope.render(
         )
 
     val customFillBrushFunction = customizations.getBrushFunction(name)
-    val customFillBrush =
+    var customFillBrush =
         if (customFillBrushFunction != null) {
             customFillBrushFunction()
         } else {
             customizations.getBrush(name)
         }
+    if (customFillBrush == null) {
+        view.shader.getOrNull()?.let {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+                val shader = RuntimeShader(it.trim().trimIndent())
+                customFillBrush = SizingShaderBrush(shader)
+                val shaderUniformTime = customizations.getShaderUniformTimeState(name)
+                if (shaderUniformTime != null) {
+                    shader.setFloatUniform("iTime", shaderUniformTime.floatValue)
+                }
+                return@let
+            } else {
+                view.shader_fallback_color.getOrNull()?.let { colorString ->
+                    customFillBrush =
+                        SolidColor(Color(android.graphics.Color.parseColor(colorString)))
```

Comment:
What happens if `parseColor()` fails? Exception?

Reply:
Yeah. It will. I changed it to use the `definition.element.Color` instead of String so it won't need to parse the color every time it draws.